CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+# v1.19.2 (2024-02-26)
+
+## OS Changes
+* Update third party packages ([#3789])
+* Update kernel to 5.10.209, 5.15.148, 6.1.77 ([#3797])
+* Add AWS settings extension ([#3738], [#3770])
+* Allow CSI helpers in the SELinux policy ([#3779])
+* Update to latest NVIDIA drivers ([#3798])
+
+## Orchestrator Changes
+
+### Kubernetes
+* Enable NVIDIA GPU isolation using volume-mounts ([#3718] thanks @chiragjn , [#3790])
+
+### ECS
+* Support for ECS_ENABLE_CONTAINER_METADATA ([#3782])
+
+## Build Changes
+* Adjust Certdog to utilize a configuration file instead of the API server ([#3706], [#3778], [#3787])
+* Use standard make instead of make macro for shim package ([#3771])
+* Renumber unit files in release package([#3769])
+* Ignore EKS patches for k8s-1.23 in Git ([#3774])
+* Clean up CNI results cache on boot ([#3792])
```

Comment:
This should go in either OS or Kubernetes changes - the k8s variants are the only ones that include the affected cni-plugins package.